miniapps/tools/load-visit-dc.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+//
+//    -------------------------------------------------------------------
+//    Load DC Miniapp:  Visualize fields saved via DataCollection classes
+//    -------------------------------------------------------------------
+//
+// This miniapp loads and visualizes (in GLVis) previously saved data using
```

Comment:
Mention this in `CHANGELOG` and `doc/CodeDocumentation.dox`?

Reply:
Done.